governance/votes/Executive Vote - September 27, 2024.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Template - [Executive Vote] SBE Reconfiguration, Sky Ecosystem Liquidity Bootstrapping, August 2024 Aligned Delegate Compensation, and Spark Proxy Spell - September 27, 2024
+summary: Smart Burn Engine reconfiguration with updated parameters, USDS and SKY Transfers for Sky Ecosystem Liquidity Bootstrapping, August MKR and DAI Aligned Delegate Compensation, and trigger Spark Proxy Spell.
+date: 2024-09-27T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] SBE Reconfiguration, Sky Ecosystem Liquidity Bootstrapping, August 2024 Aligned Delegate Compensation, and Spark Proxy Spell - September 27, 2024
+
+The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+- Smart Burn Engine (SBE) reconfiguration featuring a new contract deployment that is changing the nature of the SBE. 
```

Comment:
```suggestion
- Smart Burn Engine (SBE) will be reconfigured featuring a new contract deployment that is changing the nature of the SBE. 
```
It can also be formulated as "Reconfigure Smart Burn Engine (SBE),featuring...", but in its current form the sentence is missing the verb.